openjpa-jdbc/src/main/java/org/apache/openjpa/jdbc/schema/SchemaTool.java

```diff
@@ -1030,10 +1032,7 @@ private void drop(SchemaGroup db, SchemaGroup repos, boolean considerDatabaseSta
                             continue;
 
                         if (dropColumn(cols[k])) {
-                            if (dbTable != null)
```

Comment:
dbTable is always not null here, we have a null check 4 lines above,
I can revert if you want